client/state/current-user/gravatar-status/test/selectors.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { set } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	isCurrentUserUploadingGravatar
+} from '../selectors';
+
+describe( 'selectors', () => {
+	describe( '#isCurrentUserUploadingGravatar', () => {
+		it( 'returns false when state is undefined', () => {
+			expect( isCurrentUserUploadingGravatar( undefined ) ).to.equal( false );
+		} );
+
+		it( 'returns state when defined', () => {
+			const uploadingState = {};
+			set( uploadingState, 'currentUser.gravatarStatus.isUploading', true );
+			expect( isCurrentUserUploadingGravatar( uploadingState ) )
```

Comment:
Hmm, any lodash method that returns a new object vs mutates? I still prefer something like:
```
const state = { currentUser: { gravatarStatus: { isUploading: true } } };
expect( isCurrentUserUploadingGravatar( state ) ).to.equal( true );
```

Though I won't be a huge stickler on this, since it's not being that clever in test data.